media-libs/avisynthplus/avisynthplus-3.7.0.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="AviSynth+ scripting frameserver"
+DESCRIPTION="A powerful nonlinear scripting language for video."
+HOMEPAGE="https://avs-plus.net"
+COPYRIGHT="2000-2015 Ben Rudiak-Gould, et al., \
+2013-2021 AviSynth+ Project"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/AviSynth/AviSynthPlus/archive/v$portVersion.tar.gz"
+SOURCE_FILENAME="avisynthplus-$portVersion.tar.gz"
+SOURCE_DIR="AviSynthPlus-$portVersion"
+CHECKSUM_SHA256="8906d9e46dc90a194413d69b710e3106c0aafddc0c5c62004885d0c3beb79862"
+#PATCHES=""
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64 ?arm ?ppc"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	avisynthplus$secondaryArchSuffix = $portVersion
+	lib:libavisynth = $portVersion
+	lib:libconvertstacked = $portVersion
+	lib:libtimestretch = $portVersion
+	lib:libimageseq = $portVersion
+	lib:libshibatch = $portVersion
```

Comment:
Need to check, but are you sure these don't need $secondaryArchSuffix?

Reply:
Upon checking, it seems that they do.  I wasn't sure if that was a requirement if the package name had that variable set.

Reply:
When using secondary architecture on 32-bit cmakedirargs "should" install library in $libDir, for 32-bit this is $prefix/lib on primary, $prefix/lib/x86 on secondary architecture, hence the need for $secondaryArchSuffix